qcodes/utils/slack.py

```diff
@@ -255,13 +268,17 @@ def get_im_messages(self, username, **kwargs):
         Returns:
             List of IM messages.
         """
+        # provide backward compatibility with 'count' keyword. It still works,
+        # but is undocumented
```

Comment:
why not document it in this PR ? :)

Reply:
I'm basically leaving some code to prevent a user's code breaking that depends on an unspecified keyword argument :). It's no longer on Slack's api pages:
https://api.slack.com/methods/conversations.history

I'll make a note saying that count probably does the same as limit, with limit taking preference

Reply:
good, thank you!